package.json

```diff
@@ -20,7 +20,7 @@
     "start-prod": "npm run build-prod && npm run serve-static",
     "start-prod-readable": "npm run build-prod-readable && npm run serve-static",
     "test": "npm run build && npm run test-all",
-    "test-all": "NODE_ENV=test nyc mocha --compilers js:babel-core/register 'test/**/*.js' 'src/**/test/**/*.js'",
+    "test-all": "NODE_ENV=test nyc mocha --compilers js:babel-core/register 'src/test/**/*.js'",
```

Comment:
Maybe that's just me, but how about putting everything in `test/` instead of `src/test` ?

For instance, Mocha looks by default for `test/*.js` (but not subdirectories, so we'd still need to pass our own globbing pattern).